### ISSUE TITLE
Folder structure

### ISSUE THREAD

**Issue by mostaphaRoudsari** [ladybug-tools/honeybee-wiki#1]
What the folder structure of our wiki page for honeybee should be?

Does this need to be a place for both legacy and [+]?

Should we break them down based on energy vs daylight vs therm?

The book structure is based on Summary.md file but I think having a similar folder structure will be helpful to keep the contribution easier in the future.

I'm currently writing my notes all over the place on GitHub wiki pages and having them all in the same place here will be ideal!

We could have and still can use branches for [+] versus legacy.

**Comment by mostaphaRoudsari** [comment]
We also have the option to do something like:

legacy.ladybug.tools/honeybee-wiki and then docs.ladybug.tools/honeybee-wiki

**Comment by alexandermatthias** [comment]
It'll be important to keep legacy and plus distinct. 

If we go for a "legacy.ladybug.tools/honeybee-wiki and then docs.ladybug.tools/honeybee-wiki" solution, would the docs be for [+]? For me it seems like Plus should be in the name to make that clear. Or is docs for API documentation? 

I will give the Gitbook a go and follow up with more points of discussion in the coming days.

**Comment by mostaphaRoudsari** [comment]
Good point. Right now we are calling it [+] to distinguish it from the legacy plugins.

We can also do:

docs.ladybug.tools/honeybee-legacy-wiki and then keep docs.ladybug.tools/honeybee-wiki for [+].

I checked and unfortunately docs.ladybug.tools/legacy/honeybee-wiki is not an option.

@chriswmackey any thoughts on this?

**Comment by alexandermatthias** [comment]
One issue is that the images are much smaller in Gitbook, some are very difficult to interpret. (This can probably be overcome by adding something to each of the image links). This format seems like it would fix the image sizing problem: 
https://mostapharoudsari.gitbooks.io/ladybug-primer/content/text/components/Open_EPW_Weather_File.html?q=

The commenting system on Gitbook is really nice, and the activity tab makes it easy to get an overview of all the comments in the book. 

I tried to access Gitbook using an incognito window (to see what it would be like to access it as a non-writer) and it seems that I need to create an account with Gitbook in order to view the documentation. That could discourage people from finding it. 

Is it possible to easily output a PDF from a gitbook? (Looks like this feature is expected, but not yet here https://gitbook.canny.io/feature-requests/p/pdf-export-and-other-formats

**Comment by alexandermatthias** [comment]
Just a small housekeeping issue -- Links written as:' [Custom Schedules](Custom-Schedules)' Dont work, but links written as '[Hydra](https://hydrashare.github.io/hydra/)' Do still work.

**Comment by alexandermatthias** [comment]
RE: "Right now all the pages are credited to me which is wrong as soon as we copy the files into folders and push the new folders it will be corrected" -- Can we go over this process?

**Comment by mostaphaRoudsari** [comment]
1. We should have one landing page that refers to all the available materials. @mostaphaRoudsari to initiate and @alexandermatthias will review.
2. Legacy versus [+] goes in different branches. @mostaphaRoudsari will do this one.
3. English goes in its own branch. @mostaphaRoudsari will do this one.
4. We need to create a folder structure for the content. @mpcontreras will lead the effort.
